docs/managing-dragonfly/tiering.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 5
+---
+
+# SSD Data Tiering in Dragonfly
+
+Dragonfly v1.21.0 introduces a powerful feature: SSD data tiering. This innovative approach optimizes
+storage utilization by leveraging SSD/NVMe disks as a secondary storage tier that complements
+your existing RAM. By intelligently offloading specific data to faster disk storage,
+Dragonfly can significantly reduce physical memory usage, potentially achieving a 2-5x improvement.
+
+## How It Works
+
+Dragonfly's data tiering focuses on string values exceeding 64 characters in length.
+When enabled, these longer strings are intelligently offloaded to the SSD tier,
```

Comment:
You already used the complementing term "intelligently" above, I'd drop it here..